### DISCUSSION TITLE
window.ipcRenderer is undefined

### DISCUSSION THREAD

**Discussion by Ayush-Vish** [usebruno/bruno#648] [Q&A]
I have installed Bruno locally, and the electron app is working fine but in the next.js app the input tag to choose location is not working.
On debudding the problem I found that in the "packages\bruno-app\src\providers\ReduxStore\slices\collections\actions.js" (901) the ipcRenderer variable is not in the window object.
Please help me to solve this issue.

**Comment by helloanoop** [comment]
@Ayush-Vish This is expected.

Bruno only loads in Electron, as it needs IPC to talk to the filesystem and load the collections.

**Comment by Ayush-Vish** [comment]
@helloanoop  So how I can make collections and request in browser.

**Reply by helloanoop** [reply]
It's not possible to run Bruno on web browser alone. You have to do in electron only.
